frontend/services/entities/hosts.ts

```diff
@@ -53,63 +68,51 @@ export default {
       },
     });
   },
-  exportHosts: (options: ILoadHostsOptions | undefined) => {
-    const { HOSTS_REPORT, LABEL_HOSTS } = endpoints;
+  exportHosts: (options: IExportHostsOptions) => {
+    const { HOSTS_REPORT } = endpoints;
+    const sortBy = options.sortBy;
     const selectedLabels = options?.selectedLabels || [];
     const globalFilter = options?.globalFilter || "";
-    const sortBy = options?.sortBy || [];
     const teamId = options?.teamId || null;
     const policyId = options?.policyId || null;
-    const policyResponse = options?.policyResponse || null;
+    const policyResponse = options?.policyResponse || "passing";
     const softwareId = options?.softwareId || null;
     const visibleColumns = options?.visibleColumns || null;
+    const orderKeyParam = `?order_key=${sortBy[0].key}`;
+    const orderDirection = `&order_direction=${sortBy[0].direction}`;
 
-    let orderKeyParam = "";
-    let orderDirection = "";
-    if (sortBy.length !== 0) {
-      const sortItem = sortBy[0];
-      orderKeyParam += `&order_key=${sortItem.key}`;
-      orderDirection = `&order_direction=${sortItem.direction}`;
-    }
+    let path = `${HOSTS_REPORT}${orderKeyParam}${orderDirection}`;
 
-    let searchQuery = "";
     if (globalFilter !== "") {
-      searchQuery = `&query=${globalFilter}`;
+      path += `&query=${globalFilter}`;
     }
-
-    let path = "";
     const labelPrefix = "labels/";
 
     // Handle multiple filters
-    const label = selectedLabels.find((f) => f.includes(labelPrefix));
-    const status = selectedLabels.find((f) => !f.includes(labelPrefix));
-    const isValidStatus =
-      status === "new" || status === "online" || status === "offline";
-    if (label) {
-      const lid = label.substr(labelPrefix.length);
-      path = `${LABEL_HOSTS(
-        parseInt(lid, 10)
-      )}?${searchQuery}${orderKeyParam}${orderDirection}`;
+    const label = selectedLabels.find((f) => f.includes(labelPrefix)) || "";
+    const status = selectedLabels.find((f) => !f.includes(labelPrefix)) || "";
+    const statusFilterList = ["new", "online", "offline"];
+    const isStatusFilter = statusFilterList.indexOf(status) > 0;
```

Comment:
Just curious if there is a reason to favor `indexOf` rather than `includes`?

Reply:
Old habit :P Fixed.